apps/data/src/lib/data_repair/data_repair/repairs/_2026_04_rides_inplace.py

```
@@ -0,0 +1,95 @@
+    return out
+
+
+def _reinvert_pam_payload(value) -> str | None:
+    """Re-interleave PAM.data_raw in the VARIANT payload. Returns a JSON
+    string; the caller wraps with parse_json on the Spark side."""
+    if value is None:
+        return None
+    try:
+        raw_json = value.toJson() if hasattr(value, "toJson") else value
+        obj = json.loads(raw_json)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)